beacon_node/network/src/sync/range_sync/batch.rs

```diff
@@ -19,6 +19,33 @@ pub trait BatchConfig {
     fn max_batch_download_attempts() -> u8;
     /// The max batch processing attempts.
     fn max_batch_processing_attempts() -> u8;
+    /// Hashing function of a batch's attempt. Used for scoring purposes.
+    ///
+    /// When a batch fails processing, it is possible that the batch is wrong (faulty or incomplete) or
+    /// that a previous one is wrong. For this reason we need to re-downloaded and re-process the
+    /// unvalidated batches and the current one. Consider this scenario:
```

Comment:
```suggestion
    /// invalidated batches and the current one. Consider this scenario:
```

Reply:
This batches are not invalid, just not yet valid. Maybe `Not yet validated`?

Reply:
Ohh sorry. Yep `not yet validated` sounds good.